tools.repos

```diff
@@ -0,0 +1,5 @@
+repositories:
+  universe/tools:
```

Comment:
```suggestion
  tools:
```

I think we can keep this directly in the top level `src` folder like this.
Reason is, the core packages could use help from these tools as well.